test_data/scripts/conftest.py

```diff
@@ -0,0 +1,140 @@
+from fnmatch import fnmatch
```

Comment:
adding a conftest for two scripts to run the models and convert csv to netcdf seems to be overkill to me. The more infrastructure we add the more it has to be maintained.

Reply:
I dont expect this stuff will go much further. Adding the conftest options relieved me from maintaining separate code for handling the conus domain. The scheduler/force thing is a bit of paranoia on my part but selecting individual domains is very useful.

Conftest is part of the price i think we are paying for the abuse of pytests parallelism. But I think it's easier to use pytest+conftest this way rather than multiprocessing+argparse (or similar).

The conftest also streamlined duplicated code in the two tests and the two tests share it. 

I think we are good on balance and I dont want this stuff growing.